### DISCUSSION TITLE
Announcing ws-capacitor-secure-storage, secure storage for the web, iOS and Android

### DISCUSSION THREAD

**Discussion by aparajita** [ionic-team/capacitor#3882] [Show and tell]
I humbly present for your consideration [ws-capacitor-secure-storage](https://github.com/aparajita/ws-capacitor-secure-storage), a plugin that provides secure storage for the web, iOS and Android, with an API closely matching that of the Capacitor [Storage](https://capacitorjs.com/docs/apis/storage) plugin. If you are using the Storage plugin, this plugin is (more or less) a secure drop-in replacement.

Available via the npm repository. A demo app covering all of the plugin’s features is [here](https://github.com/aparajita/ws-capacitor-secure-storage-demo).

**Comment by distante** [comment]
Hey this looks really cool. Is there a size limit restriction?

**Reply by aparajita** [reply]
Only the limit set by the native storage mechanism, which I would imagine is larger than you could ever reach.

**Reply by aparajita** [reply]
Note that on the web, it looks like there is a 5MB per domain limit.
